book-club/ClientApp/src/App.js

```diff
@@ -65,6 +66,19 @@ export default function App() {
     //these methods populate placeholder data for now, will replace with a call to the books API
   }, [location]);
 
+  //hit books api to update global context
+  //ToDo: replace placeholder with book id from DB
+  useEffect(() => {
+    const placeholderBook = "OXzLsgEACAAJ?key=AIzaSyB8Qt63pYxZHRNxlDJQQUYQLS4Zuqcw5ZU"
+    setPageLoading(true)
+    booksAPI.getBook(placeholderBook).then((data) => {
+      //authors come back in an array, assume first is main
+      setBookAuthor(data.volumeInfo.authors[0])
+      setBookTitle(data.volumeInfo.title)
+      setPageLoading(false)
+    })
+  })
```

Comment:
Let's make sure we're always adding the optional array for useEffect to update otherwise it will run on every render cycle. Empty array is a proxy for ComponentDidMount (https://stackoverflow.com/questions/62077447/replacing-componentdidmount-with-useeffect)